src/components/ElectionJudgeList.js

```diff
@@ -17,17 +17,17 @@ class ResultsList extends Component {
 	}
 
 	displayList() {
-		if (this.state.results.length === 0) {
+		if (!this.state.results.length) {
 			return (
 				<tr>
-					<td colspan="5"><p>There are no results to display.</p></td>
+					<td colSpan="5"><p>There are no results to display.</p></td>
 				</tr>
 			);
 		}
 
-		return this.state.results.map(result => {
+		return this.state.results.map((result, index) => {
 			return (
-				<tr>
+				<tr key={`${result.PhoneNumber}--${index}`}>
```

Comment:
believe phonenumber would be sufficient here. I think there is some performance benefits of excluding index from here, but i'd have to find the resource.

Reply:
It is not -- there are people who register for multiple classes, or multiple times for the same class, so I needed the index.